package.json

```
@@ -3,9 +3,11 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "dev": "webpack",
-    "start": "webpack serve",
-    "prod": "webpack --env production"
+    "dev": "webpack --config webpack.client.js --mode development",
+    "start": "webpack-dev-server --config webpack.client.js --mode development",
+    "build": "webpack --config webpack.client.js --mode production",
+    "build:ssr": "webpack --config webpack.server.js --mode production",
+    "start:ssr": "webpack --config webpack.server.js --mode production && node ./server/main.js"
```

Present issue when you run just `start:ssr` command, application doesn't work. Webpack doesn't build client side js code and `dist/assets` folder has only svg files. It could be resolved buy small fix
```suggestion
    "start:ssr": "npm run build && npm run build:ssr && node ./server/main.js"
```

First of all you need to build client side and after that server side.
Also you could add `build:ssr` command too.